vme/etc/help/strings/clancommands.hlp

```diff
@@ -21,4 +21,6 @@ cexit            clan donate<br/>
 Appointed members:<br/>
 cwithdraw        cunstore<br/>
 <br/>
+<br/>Anyone:<br/>
```

Comment:
You should create the help page here:

https://wiki.dikumud.net/index.php?title=Help/Clancommands&action=edit&redlink=1